make-dist.sh

```diff
@@ -151,6 +154,7 @@ cp -rp h2o-docs/web/* target/docs-website/h2o-docs
 cp -p h2o-docs/src/booklets/v2_2015/source/*.pdf target/docs-website/h2o-docs/booklets
 cp -p h2o-r/R/h2o_package.pdf target/docs-website/h2o-r
 cp -rp h2o-py/docs/docs target/docs-website/h2o-py
+cp -rp h2o-r/h2o-package/docs target/docs-website/h2o-r/docs
```

Comment:
Regarding lines 156 and 157:
For Python
`cp -rp h2o-py/docs/docs target/docs-website/h2o-py`
It looks like the above results in the following link:
https://h2o-release.s3.amazonaws.com/h2o/<release>/<version>/docs-website/h2o-py/docs/index.html

Notice the `/docs/index.html`

Will `cp -rp h2o-r/h2o-package/docs target/docs-website/h2o-r/docs`
result in a `/docs/docs/index.html` instead of `/docs/index.html`?

Reply:
I've double checked that and the URL for Python is `.../docs-website/h2o-py/docs/index.html`
and for R it is `.../docs-website/h2o-r/docs/index.html` so I hope it'll be ok, but thanks for warning 🙂

Reply:
I was curious and based on what I've tried, the resulting structure depends on how the dirs are created. If we do `mkdir -p target/docs-website/h2o-py/ && cp -rp h2o-py/docs/docs target/docs-website/h2o-py/docs`, then the URL will be `.../docs-website/h2o-py/docs/index.html
and for R it is`. However if we do `mkdir -p target/docs-website/h2o-py/docs/ && cp -rp h2o-py/docs/docs target/docs-website/h2o-py/docs` then the URL will be `.../docs-website/h2o-py/docs/docs/index.html`

But because of these questions, I think it would be best to keep the R and Py docs copy command consistent.